layouts/partials/satellites-select/select.html

```diff
@@ -0,0 +1,185 @@
-.04.034-.093.06v-.073c-.053.04-.12-.034-.06-.074-.073.013-.172-.094-.1-.147-.085-.047-.039-.187-.105-.22-.04-.02-.232-.028-.225-.095.013-.127-.02-.167-.08-.267-.046-.074-.185-.154-.212-.02.073-.007.126-.074.166.033-.08-.053-.173-.02-.252.007-.086.027-.166 0-.252.02-.046.014-.35.007-.271.094-.034.013-.06-.007-.06-.047-.12.06-.212-.12-.278-.087.013.013.02.027.033.047-.06.013-.305-.067-.245.08-.06-.033-.537-.067-.497.054-.033 0-.06-.047-.08-.06-.046-.027-.046.033-.072.053-.113.067-.193-.007-.186-.107.007-.134-.027-.609.225-.549-.073-.194.378-.087.398-.2.006-.047.026-.228.006-.241-.046-.04-.112-.013-.165-.013-.08-.007-.066-.134-.066-.188a1.372 1.372 0 00-.067-.328c-.013-.046-.092-.046-.132-.073-.067-.04-.133-.067-.199-.1-.152-.074-.219-.174-.292-.321-.106-.154-.238-.275-.357-.395z"/><path d="M18.213 11.579c-.04-.08-.06-.134-.046-.188-.02-.114 0-.1-.047-.2-.02-.054-.02-.134-.06-.174-.046-.067-.026-.107-.02-.16.04-.008.087-.014.127-.008.033-.04 0-.08.046-.113.02-.06.093-.027.14-.054.039-.013-.074-.047-.06-.087.033-.02-.034-.067.026-.073.02.013.053.006.08.026.02.02.046-.02.06-.006.039.006.079-.054.118-.02.027.026.06.06.053.107.007.046.086.013.1.06.046.013.106.06.145.027.02-.034.04.04.073-.014.093-.006.067.127.047.174-.033.007-.053-.02-.113-.02-.053-.013-.06-.007-.126-.027-.06-.013-.08-.02-.139-.04-.053-.006-.06-.02-.12-.027-.052.007-.046.014-.105.014-.093.033-.08.027-.126.073-.027.06-.046.067-.046.134.066.04.092.02.106.08.006.04-.034.04-.027.087.086.014.1.02.186.04-.027.034-.047.047-.1.094-.033.074.047.094.093.167.013.04-.007.087-.04.148.113.12.212.127.18.214-.1.026-.12.006-.206-.02-.033-.027-.033-.054-.047-.06-.033-.04-.073-.048-.1-.054-.045-.06-.012-.04-.052-.1zM47.792 31.277c-.033.02-.02.034.027.047-.013-.02-.02-.033-.027-.047zM47.885 30.127c-.04.013-.046.06-.033.094a.16.16 0 01.033-.094zM47.952 30.267c-.034-.006-.034.007-.007.047.007-.02.007-.033.007-.047zM48.19 29.699c-.053.013-.047.12.013.107.033-.013 0-.094-.013-.107zM48.296 29.98c0 .006-.006.02-.006.027.033 0 .06-.007.092-.02-.033 0-.06-.007-.086-.007zM48.409 29.538c0 .067.112.154.165.08.04-.053-.132-.08-.165-.08zM48.462 30.047c-.007.013-.007.026-.007.047.113.006.212.066.318.093.046.007.093.034.146.02.053-.013.112-.047.172-.033-.046-.067-.166-.067-.238-.074a3.882 3.882 0 00-.391-.053zM49.27 33.619c-.007.013-.007.026-.014.046.093-.033.186-.013.279-.02.092 0 .145-.047.212-.1.033-.027.225-.26.185-.308-.113-.14-.185-.154-.351-.08-.172.067-.239.301-.311.462z"/><path d="M49.807 33.652c.013-.007.026-.007.046-.013-.02-.047-.02-.134-.093-.1-.1.04-.02.113.047.113zM49.787 32.763c0 .033.027.06.06.066-.007-.033-.027-.06-.06-.066z"/><path d="M46.507 29.726a.083.083 0 00-.02-.094c.14-.033.252-.04.358-.14.1-.094.199-.134.331-.1.133.033.199.133.318.18.12.053.239-.014.345.067-.086.06-.007.147.033.207.046.087-.007.154-.04.234-.08.2.04.442.126.616.033-.047-.013-.148.04-.181.073-.047.06.16.06.18 0 .04.013.047.04.074.039.04.019.087.052.127.066.074.206.114.278.034a2.03 2.03 0 01.173-.168c.06-.06.072-.14.152-.194.073-
```

Comment:
create 1 function selectSatellite with the name parameter. Start with creating table object with needed constants